server/src/schema.rs

```diff
@@ -328,6 +456,29 @@ table! {
     }
 }
 
+table! {
+    user_fast (fast_id) {
+        id -> Nullable<Int4>,
+        actor_id -> Nullable<Varchar>,
+        name -> Nullable<Varchar>,
+        avatar -> Nullable<Text>,
+        email -> Nullable<Text>,
+        matrix_user_id -> Nullable<Text>,
+        bio -> Nullable<Text>,
+        local -> Nullable<Bool>,
+        admin -> Nullable<Bool>,
+        banned -> Nullable<Bool>,
+        show_avatars -> Nullable<Bool>,
+        send_notifications_to_email -> Nullable<Bool>,
+        published -> Nullable<Timestamp>,
+        number_of_posts -> Nullable<Int8>,
```

Comment:
ditto re: normalized data vs counting stats.  As an example, we could have `user_stats` with the below columns and `user_id` and just join to `user` to get the name, email, etc